### ISSUE TITLE
Training procedure

### ISSUE THREAD

**Issue by CharlesGong12** [Picsart-AI-Research/PAIR-Diffusion#15]
Hi, thanks for your amazing work!
I find your work so interesting and I wanna train the models by myself. Could you provide training code?

**Comment by vidit98** [comment]
Hi, thanks for your interest! It would be difficult to release the training procedure soon. But I can help if you have any specific doubts. The training script would be very similar to [LDM](https://github.com/CompVis/latent-diffusion). The major change would be in dataloader where you need to read the segmentation mask as well.
